zaza/openstack/charm_tests/rabbitmq_server/tests.py

```diff
@@ -26,8 +26,8 @@
 import zaza.openstack.charm_tests.test_utils as test_utils
 import zaza.openstack.utilities.generic as generic_utils
 
-from charmhelpers.core.host import CompareHostReleases
```

Comment:
I'm surprised, see my other question, why did we copy this code from charm-helpers in here?

Reply:
Oh that's deliberate, but perhaps a tad conscientious.  I don't think that zaza-openstack-tests should rely on charmhelpers as a library.  charmhelpers is really meant for running on the unit, and this is a test framework.  CompareHostReleases is a relatively small piece of code and the list of releases (as a concept) is already in os_versions.py in various forms.

Therefore, it actually feels better to drop charmhelpers as a dependency for z-o-t just for this one function (which is useful) and just copy the code in.

Reply:
Makes sense, thanks!